### PR TITLE
Add modal that tells users about the user page.

### PR DESCRIPTION
**Pull request by jagtalon**
screen shot 2016-05-30 at 4 14 08 pm

### CODE REVIEW

**Comment by MariagraziaAlastra** [comment]
LGTM :+1:

**Comment by MariagraziaAlastra** [comment]
@jagtalon Who should see this before it's merged?

File: root/static/js/userpages/app.js [hunk 1]
```diff
@@ -7,7 +7,9 @@
 'use strict';
 
 // var app = angular.module('app', ['ngSanitize', 'textAngular', 'flow']);
-var app = angular.module('app', ['ngSanitize']);
+var app = angular.module('app', ['ngSanitize']).config(function($locationProvider) {
+    $locationProvider.html5Mode({enabled: true, requireBase: false});
+    })
```
Comment (jbarrett) [inline root/static/js/userpages/app.js]: Going to guess this was the redirect code?

Comment (jagtalon) [inline root/static/js/userpages/app.js]: @jbarrett this is actually the part that prevents the weird slash-before-the-hash thang.

**Comment by jagtalon** [comment]
@MariagraziaAlastra It has been seen by @zekiel and @russellholt as well.

**Comment by MariagraziaAlastra** [comment]
Thanks @jagtalon!

**Pull request merged**